mne/epochs.py

```diff
@@ -2582,6 +2733,15 @@ class Epochs(BaseEpochs):
     For example with the event_id ``{'aud': 1, 'vis': 2}`` and the events
     ``[[0, 0, 1], [0, 0, 2]]``, the "merge" behavior will update both event_id
     and events to be: ``{'aud/vis': 3}`` and ``[[0, 0, 3]]`` respectively.
+
+    There is limited support for :class:`~mne.Annotations` in the
+    :class:`~mne.Epochs` class. Currently annotations that are present in the
+    :class:`~mne.io.Raw` object will be preserved in the resulting
+    :class:`~mne.Epochs` object, but it is not yet possible to add annotations
+    to the Epochs object programmatically (via code) or interactively (through
+    the plot window). Additionally, concatenating :class:`~mne.Epochs` objects
+    that contain annotations is not supported, and any annotations will be
+    dropped when concatenating.
```

Comment:
An enumerated list should help readability, and I think the third point (which to me is the most important next step) was missing
```suggestion
    There is limited support for :class:`~mne.Annotations` in the
    :class:`~mne.Epochs` class. Currently annotations that are present in the
    :class:`~mne.io.Raw` object will be preserved in the resulting
    :class:`~mne.Epochs` object, but:
    
    1. It is not yet possible to add annotations
       to the Epochs object programmatically (via code) or interactively
       (through the plot window)
    2. Concatenating :class:`~mne.Epochs` objects
       that contain annotations is not supported, and any annotations will
       be dropped when concatenating.
    3. Annotations will be lost on save.
```

Reply:
Fair point :p. Will eliminate the 3rd point on the short follow-up PR.